src/compiler/objective_c_plugin.cc

```diff
@@ -81,6 +108,13 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
   ObjectiveCGrpcGenerator() {}
   virtual ~ObjectiveCGrpcGenerator() {}
 
+ private:
+  static const ::grpc::string kNonNullBegin = "NS_ASSUME_NONNULL_BEGIN\n";
```

Comment:
For the compiler errors, I'm fine with putting these static strings in file scope.